packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -196,6 +254,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     <>
       <Head>
         <link rel="preload" as="image" href={post?.image} />
+        {schemaComponent}
```

Comment:
We can also maybe utilize https://github.com/garmeeh/next-seo?tab=readme-ov-file#article-1

Reply:
It is unclear to me if that library supports all of the parameters we exposed here. But definitely worth a look.

Reply:
Came here to comment the same on using this 😅

Reply:
From looking at the source, they have predefined some JSON-LD keys for article, and then just accepts `...rest`.

So I think it takes all

https://github.com/garmeeh/next-seo/blob/96b49b63caae776d3f459b7bb1b49d6cd416a241/src/jsonld/article.tsx#L50

https://github.com/garmeeh/next-seo/blob/96b49b63caae776d3f459b7bb1b49d6cd416a241/src/jsonld/jsonld.tsx#L30

Reply:
I can change this, but I am just curious to understand why we are pushing for using a library for something relatively simple for which we already have a pretty clear idea of how to implement. I guess I don't see the advantage of using `next-seo` for this, almost feels like an `isOdd` library, but maybe it's just me 😅

Reply:
We already have next-seo library :) 
Kind of neater implementation atleast this is my opinion

Reply:
I don't personally have any strong preference, looks fine to me the way it is, just maybe that library makes it more beautiful to the eye

Reply:
my concern with these things is they obfuscate the underlying API, and you end up needing to lookup 2 documentations for the same thing, the official schema, and then the docs of next-seo to see how they are expecting that schema to be attached to their components. 

I'll follow the majority here, but in general my 2 cents on third-party libraries is to only use them if you really need to

Reply:
Agree on being basic but what I think this should end up as a our component either way so then it does not matter what we use under the hood (next-seo or our implementation) it should just accept the `post` and we can change implementation under the hood any time.